package.json

```diff
@@ -69,7 +69,8 @@
     "testPathIgnorePatterns": [
       "/node_modules/",
       "/dist/",
-      "fixtures[.].*[.]ts"
+      "fixtures[.].*[.]ts",
+      "testUtils.findTiles.ts"
```

Comment:
Nitpick: note that this is a regex so the correct form would be `"testUtils[.]findTiles[.]ts"`.

It might also make sense to make this generic? For example `"testUtils[.].*[.]ts"`?

Fftr either way.